scripts/talents/talent-tree-types.ts

```diff
@@ -0,0 +1,103 @@
+export enum ResourceTypes {
+  Mana,
+  Rage,
+  Focus,
+  Energy,
+  ComboPoints,
+  Runes,
+  RunicPower,
+  SoulShards,
+  LunarPower,
+  HolyPower,
+  AlternatePower,
+  Maelstrom,
+  Chi,
+  Insanity,
+  Unused14PowerType,
+  Unused15Powertype,
+  ArcaneCharges,
+  Fury,
+  Pain,
```

Comment:
If this is for Vengeance DH, they actually use Fury now.

Reply:
putput is just expecting all the resource types from the database. If its being used or note isn't really that important :P